modules/admin_manual/pages/release_notes.adoc

```diff
@@ -24,6 +25,205 @@
+// TODO 
+* Two new config.php options (`proxy` and `proxyuserpwd`) have been added to enable federation when an instance needs to go through an authenticated proxy to reach a federated instance. See `config.sample.php` and the **LINK** [ownCloud Documentation] for more information. https://github.com/owncloud/core/pull/35868[#35868]
+
+https://github.com/owncloud/docs/issues/1869
+
+* The `occ files:scan` command is now case-insensitive for the userid https://github.com/owncloud/core/pull/35324[#35324]
+* A new `config.php` option (`dav.enable.tech_preview`) has been added to disable tech preview APIs by default https://github.com/owncloud/core/issues/36124[#36124]
+* [PHOENIX] Support for redirecting links to ownCloud Phoenix frontend has been added by introducing a new `config.php` option which stores the address where Phoenix is reachable (e.g., `'phoenix.baseUrl' => 'http://phoenix.example.tld:port'`) https://github.com/owncloud/core/pull/35819[#35819]
+* When loading groups, the performance of users has been improved https://github.com/owncloud/core/pull/35822[#35822]
+* Memory handling for the trashbin expiry background job has been improved  https://github.com/owncloud/core/pull/35708[#35708]
+
+=== Solved known issues
+
+* A new occ command, `encryption:fix-encrypted-version`, has been introduced to address issues related to encrypted files no longer being accessible. 
+  This originated from a security measure to avoid that encrypted files with the same content look identical. 
+  In some cases, users get a `Bad Signature` error when trying to access files. 
+  The new command corrects this behavior, making files accessible again. 
+  The command only needs to be run if users report the mentioned error. 
+  https://github.com/owncloud/encryption/pull/115[#115]
+* If an instance uses the `share_folder` config.php option to gather incoming user shares in a specific folder, this folder cannot be deleted by users anymore. 
+  https://github.com/owncloud/core/pull/35998[#35998]
+* The `share_folder` `config.php` option now also respects federated shares. 
+  https://github.com/owncloud/core/pull/35396[#35396]
+* The `user.min_search_length` config.php option now also respects federated users. 
+  https://github.com/owncloud/core/pull/35977[#35977]
+* Issues with database conversions using the `db:convert-type` occ command (e.g., SQLite to MySQL) have been fixed. 
+  This is still in an experimental state and should be tested thoroughly. 
+  Please provide feedback if you encounter issues. 
+  https://github.com/owncloud/core/pull/35390[#35390]
+* *File integrity checking has been improved to prevent issues:* If a checksum mismatch occurs after uploading a file, the uploaded file and its checksum is deleted to prepare for a clean re-upload.
```

Comment:
is there a reason for making some entries bold and some not? @settermjd

Reply:
The ones that were bolded were done based on how you wrote those parts of the content.